src/Language/PureScript/Make/Actions.hs

```diff
@@ -269,7 +271,7 @@ buildMakeActions outputDir filePathMap foreigns usePrefix =
   requiresForeign = not . null . CF.moduleForeign
 
   progress :: ProgressMessage -> Make ()
-  progress = liftIO . putStrLn . renderProgressMessage
+  progress = liftIO . TIO.hPutStr stderr . (<> "\n") . renderProgressMessage
```

Comment:
Is there a reason for not using `TIO.putStrLn`?

Reply:
Perhaps a comment here to explain why we aren't using `hPutStrLn` would be good?

Reply:
Ah oops! Sorry for the redundant conversation 😄